server/src/controllers/userControllers.ts

```diff
@@ -140,6 +140,19 @@ export const getUser = async (
   }
 };
 
+export const getUserById = async (
+  req: IReqAuth,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const user = await User.findById(req.params.id);
```

Comment:
We will need to ensure the user's token information is sanitized

Reply:
Yes can you please, I wasn't sure how to go about this.